packages/block-editor/src/components/floating-toolbar/index.native.js

```diff
@@ -0,0 +1,81 @@
+/**
```

Comment:
It's not a blocker but I think this file should be moved instead of copied. In the current approach, we lose history.